schema/annotation-source.yaml

```diff
@@ -36,14 +36,14 @@ $defs:
         description: >-
           A quantitative score reflecting the degree of confidence that the information 
           the information entity represents is true.
-      method:
+      specified_by:
         oneOf:
         - $ref: "#/$defs/Method"
         - $ref_curie: gks.core:CURIE
         description: >-
           A :ref:`Method` that describes all or part of the process through which the information was
           generated.
-      contributions:
+      contribution:
```

Comment:
Changed to name used in VA model, where convention is for attribute names to be singular (but don't feel strongly).

Reply:
It would be helpful to be consistent across vrs, vrsatile and va to be all singular or always plural when multiples are allowed.

@mbrush i think we use plurals in VRS (eg Haplotype.members) so it would be less disruptive to go that way.

Reply:
Sure - see my reply to Alex about this above.